record_tx_strategy_external_incoming_tx.go

```diff
@@ -122,33 +122,33 @@ func _hydrateExternalWithSync(tx *Transaction) {
 		tx.GetOptions(true)...,
 	)
 
-	// to simplfy: broadcast every external incoming txs
+	// to simplify: broadcast every external incoming txs
 	sync.BroadcastStatus = SyncStatusReady
 
 	sync.P2PStatus = SyncStatusSkipped  // the sender of the Tx should have already notified us
-	sync.SyncStatus = SyncStatusPending // wait until transaciton will be broadcasted
+	sync.SyncStatus = SyncStatusPending // wait until transactions will be broadcasted
 
 	// Use the same metadata
 	sync.Metadata = tx.Metadata
 	sync.transaction = tx
 	tx.syncTransaction = sync
 }
 
-func _externalIncomingBroadcast(ctx context.Context, logger zLogger.GormLoggerInterface, tx *Transaction, allowErrors bool) error {
-	logger.Info(ctx, fmt.Sprintf("ExternalIncomingTx.Execute(): start broadcast, TxID: %s", tx.ID))
+func _externalIncomingBroadcast(ctx context.Context, logger *zerolog.Logger, tx *Transaction, allowErrors bool) error {
+	logger.Info().Msgf("ExternalIncomingTx.Execute(): start broadcast, TxID: %s", tx.ID)
 
 	err := broadcastSyncTransaction(ctx, tx.syncTransaction)
 
 	if err == nil {
 		logger.
-			Info(ctx, fmt.Sprintf("ExternalIncomingTx.Execute(): broadcast complete, TxID: %s", tx.ID))
+			Info().Msgf("ExternalIncomingTx.Execute(): broadcast complete, TxID: %s", tx.ID)
 
 		return nil
 	}
 
 	if allowErrors {
 		logger.
-			Warn(ctx, fmt.Sprintf("ExternalIncomingTx.Execute(): broadcasting failed, next try will be handled by task manager. Reason: %s, TxID: %s", err, tx.ID))
+			Warn().Msgf("ExternalIncomingTx.Execute(): broadcasting failed, next try will be handled by task manager. Reason: %s, TxID: %s", err, tx.ID)
```

Comment:
`ExternalIncomingTx.Execute()` Can be added to `Str()`

Reply:
Removed function name from logs